lamindb_setup/_schema_metadata.py

```diff
@@ -156,19 +157,23 @@ class FieldMetadata(BaseModel):
 
 class _ModelHandler:
     def __init__(self, model, module_name: str, included_modules: list[str]) -> None:
+        from lamindb.models import LinkORM
+
         self.model = model
         self.class_name = model.__name__
         self.module_name = module_name
         self.model_name = model._meta.model_name
         self.table_name = model._meta.db_table
         self.included_modules = included_modules
         self.fields = self._get_fields_metadata(self.model)
+        self.is_link_table = issubclass(model, LinkORM)
```

Comment:
I thought we had this line already since the beginning. Good that we track this now!

Reply:
We had this at the field level, to know if a field comes from a link table, but not at the model level.